packages/directives/trap-focus/index.ts

```diff
@@ -1,18 +1,17 @@
-// @ts-nocheck
 import { nextTick } from 'vue'
-import { obtainAllFocusableElements, off, on } from '@element-plus/utils'
+import { obtainAllFocusableElements } from '@element-plus/utils'
 import { EVENT_CODE } from '@element-plus/constants'
 import type { ObjectDirective } from 'vue'
 
 export const FOCUSABLE_CHILDREN = '_trap-focus-children'
 export const TRAP_FOCUS_HANDLER = '_trap-focus-handler'
 
-export interface ITrapFocusElement extends HTMLElement {
+export interface TrapFocusElement extends HTMLElement {
```

Comment:
## Etc

I think changing the name of the interface you export can affect users. What do you think?

Reply:
I don't think so, It seems to only be used inside `element-plus`.

Reply:
Ah, thinking about it again, it seems that the component is only used internally. You're right.